plugins/generator-1.15.2/forge-1.15.2/block.definition.yaml

```diff
@@ -333,6 +333,30 @@ templates:
     writer: json
     name: "@MODASSETSROOT/models/block/@registryname_down.json"
 
+  # TX: Wood StoneButton templates
+  - template: json/txblock/button_item.json.ftl
+    condition: "blockBase %= Button"
+    writer: json
+    name: "@MODASSETSROOT/models/item/@registryname.json"
+  - template: json/txblock/button_states.json.ftl
+    condition: "blockBase %= Button"
+    writer: json
+    name: "@MODASSETSROOT/blockstates/@registryname.json"
+  - template: json/txblock/button.json.ftl
+    condition: "blockBase %= Button"
+    writer: json
+    name: "@MODASSETSROOT/models/block/@registryname_button.json"

```

Comment:
There's an extra "_button" in here and in the pressed model name (same for 1.16)